src/others/table-formats.tsx

```diff
@@ -0,0 +1,11 @@
+import { CellProps } from 'react-table';
+import { MenuDocument } from 'model/menu/menu';
+import { UnitOfMeasure } from 'model/enums/unit-of-measure';
+
+type UnitOfMeasureKey = keyof typeof UnitOfMeasure;
+
+export const currency = ({ value }: CellProps<MenuDocument>): JSX.Element => <div> {`${value} zł`} </div>;
```

Comment:
I think using the `<Text />` component from Chakra would be more semantic, rather than placing text inside `<div>`.

Reply:
You are right, I changed that.